WebRoot/js/zimbraMail/abook/view/ZmContactSearch.js

```diff
@@ -428,23 +418,46 @@ ZmContactSearchListView.prototype._getHeaderList =
 function() {
 	var headerList = [];
 	headerList.push(new DwtListHeaderItem({field:ZmItem.F_TYPE, width:ZmMsg.COLUMN_WIDTH_FOLDER_CN}));
-	headerList.push(new DwtListHeaderItem({field:ZmItem.F_NAME, text:ZmMsg._name, width:ZmMsg.COLUMN_WIDTH_NAME_CN}));
-	headerList.push(new DwtListHeaderItem({field:ZmItem.F_EMAIL, text:ZmMsg.email}));
-
+	headerList.push(new DwtListHeaderItem({field:ZmItem.F_NAME, text:ZmMsg._name}));
 	return headerList;
 };
 
+// Overriding the DwtListView.prototype.createHeaderHtml as we donot need a header, here.
+
```

Comment:
I have done that for readability. 
Moreever, in our app, we have similar patterns.
https://github.com/Zimbra/zm-web-client/blob/feature/universal-ui/WebRoot/js/zimbraMail/mail/ZmMailApp.js#L131

https://github.com/Zimbra/zm-web-client/blob/feature/universal-ui/WebRoot/js/zimbraMail/mail/ZmMailApp.js#L1020

https://github.com/Zimbra/zm-web-client/blob/feature/universal-ui/WebRoot/js/zimbraMail/mail/ZmMailApp.js#L1576